IFC4x3/Templates/Object Connectivity/Station/DocTemplateDefinition.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
My IfcDoc is crashing when looking at this one.

Reply:
[1c8e3b2](https://github.com/bSI-InfraRoom/IFC-Specification/pull/557/commits/1c8e3b21cbc0f1ed5945da4688c82bcc5fc4f222)

not sure what happened. Pull and try again.

Reply:
This one now works inside IfcDoc. Issues still to decide:
- "Hierarchical tree of linear positioning elements"
- Usages: PropertySets for Objects (IfcReferent and IfcLinearPositioningElement)